integration_test_lite.py

```diff
@@ -31,32 +31,32 @@ def __init__(self):
         session = self.session
         config_json = self.config_json
         post_data = {
-            'client_id': config_json["client_id"],
-            'client_secret': config_json["client_secret"],
+            'client_id': config_json['client_id'],
+            'client_secret': config_json['client_secret'],
             'grant_type': 'client_credentials'
         }
 
-        request = await session.post(config_json["token_api"], data=post_data)
-        response = await request.json()
-        self.access_token = f'Bearer {response["access_token"]}'
+        response = await session.post(config_json['token_api'], data=post_data)
+        json = await response.json()
```

Comment:
Since `json` was imported as a module, can you rename this variable for clarity?